projects/vdk-plugins/vdk-dag/src/vdk/plugin/dag/local_executor.py

```diff
@@ -158,9 +159,10 @@ def __find_job_path(job_name: str):
         candidates = [
             os.getcwd(),
         ]
+        # TODO: expose this using the plugin configuration mechanisms (which infers also from env. vars among others)
         candidates += [
             part
-            for part in os.environ.get("DAG_LOCAL_RUN_JOB_PATH", "").split(":")
+            for part in os.environ.get("DAGS_LOCAL_RUN_JOB_PATH", "").split(":")
```

Comment:
was this a bug previously or was the var name changed here accidentally; I don't see any other corrections to the var name elsewhere

Reply:
Renamed DAG_LOCAL_RUN_JOB_PATH to DAGS_LOCAL_RUN_JOB_PATH to bring it inline with other configuration options [like here](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-dag/src/vdk/plugin/dag/dag_plugin_configuration.py#L6). Eventually we should expose it the same way but not in this PR.